src/WebJobs.Script.WebHost/Filters/AuthorizationLevelAttribute.cs

```diff
@@ -55,7 +56,11 @@ public async override Task OnAuthorizationAsync(HttpActionContext actionContext,
             // If the request has not yet been authenticated, authenticate it
             if (requestAuthorizationLevel == AuthorizationLevel.Anonymous)
             {
-                string armToken = GetArmTokenHeader(actionContext);
+                string armToken = null;
+                if (FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagSwtAuthenticationEnabled))
```

Comment:
Note that v1 doesn't support hosting configuration like v4/v3. We only have per-application feature flags.